examples/Python/Basic/pointcloud.ipynb

```diff
@@ -241,6 +241,37 @@
     "`paint_uniform_color` paints all the points to a uniform color. The color is in RGB space, [0, 1] range."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pointcloud Distance\n",
+    "Open3D provides the method `compute_point_cloud_distance` to compute the distance from a source pointcloud to a target pointcloud. I.e., it computes for each point in the source pointcloud the distance to the closest point in the target point cloud.\n",
+    "\n",
+    "In the example below we use the function to compute the difference between to pointclouds. Note that this method could also be used to compute the Chamfer distance between two pointclouds."
```

Comment:
`pointcloud` -> `point cloud` to be consistent with the rest.
`compute the difference between to ...` -> `compute the difference between two ...`

Reply:
done